rotkehlchen/exchanges/bitstamp.py

```diff
@@ -614,8 +621,8 @@ def _get_trade_pair_data_from_transaction(raw_result: Dict[str, Any]) -> TradePa
         # storing the raw pair strings.
         base_asset_symbol, quote_asset_symbol = pair.split('_')
         try:
-            base_asset = Asset(base_asset_symbol)
-            quote_asset = Asset(quote_asset_symbol)
+            base_asset = asset_from_bitstamp(base_asset_symbol)
+            quote_asset = asset_from_bitstamp(quote_asset_symbol)
```

Comment:
you don't need to handle the deseriazliation error here, since `get_trade_pair_data_from_transaction` already may raise it. So let it propagate upwards.